.github/workflows/test.yml

```diff
@@ -23,8 +23,11 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: 14.x
-      
+          node-version: 15.x
```

Comment:
Ran into a bug cloning the repo, even though it's public. NPM 7 does away with that restriction.

Read [here](https://github.com/actions/setup-node/issues/214#issuecomment-1014695382) that NPM 7 fixes the bug, then read [here](https://github.com/actions/setup-node/issues/213#issuecomment-739685886) that upping node to 15 will get us that.

Reply:
Oh yeah, you can upgrade it to node 16, the node version shouldn't really matter for the CI, as long it does it's job it's fine. We don't depend on specific node versions as a browser extension.

Reply:
Ok, if the rollup plugin upgrade request is ignored, I'll circle back.

Reply:
On the other hand, 16 is the stable version. But I'm it would no longer be "in scope" (aka annoying me) :-).